### DISCUSSION TITLE
中国民航大学的美食有哪些

### DISCUSSION THREAD

**Discussion by maning711** [CollegesChat/university-information#268] [Enhancement]
中国民航大学的美食有哪些

**Comment by hud86n** [comment]
24年上半年食堂改造了，北一二楼味道还可以，好吃的档口集中在北二食堂二楼。南一食堂承包给厦门航空，一楼二楼都不好吃，三楼不知道还开不开，好吃的集中在南二二楼和三楼，但是距离稍微远一点。
